src/main/kotlin/no/skatteetaten/aurora/cantus/service/ImageTagType.kt

```diff
@@ -0,0 +1,29 @@
+package no.skatteetaten.aurora.cantus.service
+
+enum class ImageTagType {
```

Comment:
Er denne koden kopiert fra gobo? Kanskje vi skulle vurdere å lage et lite lib for dette? Reglene for parsing av versjoner er faktisk ganske sentral, og vi vil da ha et naturlig sted å dokumentere dette for tredjeparter/brukere.

Reply:
Ja det stemmer at denne er tatt fra Gobo med litt fjerning av metoder Cantus ikke trenger. Men ja det kan være lurt å lage et lib for dette. Er det flere enn Gobo og Cantus som bruker dette?

Reply:
Mulig dette burde flyttes ut som @bentsolheim nevnte, men jeg tror også at de ulike casene her burde ha enhetstester. Jeg ser at noen av tilfellene testes igjennom andre tester, men flere av dem blir ikke testet. Dette er vel en fin kandidat for en `@ParameterizedTest`?